src/Sylius/Bundle/AdminBundle/Console/Command/ListAdminUsersCommand.php

```diff
@@ -0,0 +1,120 @@
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\AdminBundle\Console\Command;
+
+use Sylius\Component\Core\Model\AdminUser;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Sylius\Component\Core\Model\AdminUserInterface;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Sylius\Component\User\Repository\UserRepositoryInterface;
+
+#[AsCommand(
+    name: 'sylius:admin-user:list',
+    description: 'Displays all admin users with information in a list. Optionally, also one user can be displayed if the username is specified as an option.',
+)]
+final class ListAdminUsersCommand extends Command
+{
+    protected SymfonyStyle $io;
+
+    /** @param UserRepositoryInterface<AdminUserInterface> $userRepositoryInterface */
+    public function __construct(
+        private readonly UserRepositoryInterface $userRepositoryInterface
+    ) {
+        parent::__construct();
+    }
+
+    protected function initialize(InputInterface $input, OutputInterface $output): void
+    {
+        $this->io = new SymfonyStyle($input, $output);
+    }
+
+    /**
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     * @return int
+     */
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $this->io->title('List available admin users');
+
+        $username = $input->getOption('username');
+        if (isset($username)) {
+            $user = $this->userRepositoryInterface->findOneBy(['username' => $username]);
+            if ($user === null)
+            {
+                $this->io->error(sprintf('The user with the username "%s" has not been found', $username));
+                return Command::FAILURE;
+            }
+            $this->listSingleAdminUser($user);
+        } else {
+            $this->listAllAdminUsers();
+        }
+
+        return Command::SUCCESS;
+    }
+
+    private function listAllAdminUsers(): void
+    {
+        $adminUsers = $this->userRepositoryInterface->findAll();
+        /** @var AdminUser $adminUser */
+        foreach ($adminUsers as $adminUser) {
+            $this->io->table(
+                [
+                    'ID', 'E-Mail', 'Username', 'First name', 'Last name', 'Locale code', 'Enabled',
+                ],
+                [
+                    [
+                        $adminUser->getId(),
+                        $adminUser->getEmail(),
+                        $adminUser->getUsername(),
+                        $adminUser->getFirstname() ?? 'No Firstname Set',
+                        $adminUser->getLastName() ?? 'No Lastname Set',
+                        $adminUser->getLocaleCode(),
+                        $adminUser->isEnabled() ? 'Enabled' : 'Disabled',
+                    ],
+                ],
+            );
```

Comment:
You could call the `::listSingleAdminUser` here instead. It seems the content is the same 👋